src/docs/asciidoc/coreFeatures.adoc

```diff
@@ -823,7 +805,7 @@ that is similar in concept to a `Processor` in the sense that it lets you
 populate the sequence with data or terminate it).
 
 If, after exploring the above alternatives, you still think you need a
-`Processor`, head to the <<processor>> appendix to learn about the different
+`Processor`, to the <<processor>> appendix to learn about the different
```

Comment:
missing a word here, used to be `head to` but now the phrase doesn't make sense ⚠️